packages/compass-context-menu/src/context-menu-provider.tsx

```diff
@@ -63,11 +69,14 @@ export function ContextMenuProvider({
         return;
       }
 
+      if (onContextMenuOpen) {
+        onContextMenuOpen(itemGroups);
+      }
```

Comment:
Nit, for consistency with other code in this patch

```suggestion
      onContextMenuOpen?.(itemGroups);
```